sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -676,6 +676,64 @@ async def test_async_queue_by_queue_client_conn_str_receive_handler_receiveandde
                 peeked_msgs = await receiver.peek_messages(max_message_count=10, timeout=10)
                 assert len(peeked_msgs) == 0
 
+    @pytest.mark.asyncio
+    @pytest.mark.liveTest
+    @pytest.mark.live_test_only
+    @CachedServiceBusResourceGroupPreparer(name_prefix="servicebustest")
+    @CachedServiceBusNamespacePreparer(name_prefix="servicebustest")
+    @ServiceBusQueuePreparer(
+        name_prefix="servicebustest", dead_lettering_on_message_expiration=True, lock_duration="PT10S"
+    )
+    @pytest.mark.parametrize("uamqp_transport", uamqp_transport_params, ids=uamqp_transport_ids)
+    @ArgPasserAsync()
+    async def test_async_queue_receive_handler_peek_check_uniq_corr_id(
+        self, uamqp_transport, *, servicebus_namespace=None, servicebus_queue=None, **kwargs
+    ):
+        fully_qualified_namespace = f"{servicebus_namespace.name}{SERVICEBUS_ENDPOINT_SUFFIX}"
```

Comment:
is peek the only mgmt operation we want to check, what about auto lock renewer? we could maybe add that to perf?

Reply:
actually if we haven't already can we see what perf impact this has, maybe it will improve all mgmt ops?!

Reply:
yeah I was planning on adding an alr test to the follow up lock pending op remove PR, b/c that one is affects alr renews

Reply:
Will run the perf on that one as well, and merge this PR into that one: #39633